Maps.hooks.php

```diff
@@ -10,6 +10,123 @@
  */
 final class MapsHooks {
 
+ 	public static function onExtensionCallback() {
+		if ( defined( 'Maps_COORDS_FLOAT' ) ) {
+			// Do not initialize more than once.
+			return 1;
+		}
+
+		// The different coordinate notations.
+		define( 'Maps_COORDS_FLOAT' , 'float' );
+		define( 'Maps_COORDS_DMS' , 'dms' );
+		define( 'Maps_COORDS_DM' , 'dm' );
+		define( 'Maps_COORDS_DD' , 'dd' );
+
+		require_once __DIR__ . '/Maps_Settings.php';
+
+		define( 'Maps_VERSION' , '4.2.1' );
+		define( 'SM_VERSION', Maps_VERSION );
+
+		// Only initialize the extension when all dependencies are present.
+		if ( !defined( 'Validator_VERSION' ) ) {
+			throw new Exception( 'You need to have Validator installed in order to use Maps' );
+		}
+
+		$GLOBALS['egMapsStyleVersion'] = $GLOBALS['wgStyleVersion'] . '-' . Maps_VERSION;
+	}
+
+	public static function onExtensionFunction() {
+		if ( $GLOBALS['egMapsDisableExtension'] ) {
+			return true;
+		}
```

Comment:
Sounds like this again is for compat reasons. You can drop support for MW < 1.27.